lib/Gateway.js

```diff
@@ -11,26 +13,46 @@ const hassCfg = reqlib('/hass/configurations.js')
 const hassDevices = reqlib('/hass/devices.js')
 const version = reqlib('package.json').version
 
-debug.color = 2
+const STORE_DIR = reqlib('config/app.js').storeDir
+const CUSTOM_DEVICES = 'customDevices.json'
 
-try {
-  const storeDir = reqlib('config/app.js').storeDir
-  const customDevices = reqlib(storeDir + '/customDevices')
-  let i = 0
-  for (const d in customDevices) {
-    hassDevices[d] = customDevices[d]
-    i++
+// Continue supporting customDevices.js for now
+if (fs.existsSync(path.join(STORE_DIR, 'customDevices.js'))) {
```

Comment:
I think we could remove this and keep just the `loadDevices` method. Add a var  `this.customDevicesLength` in Gateway constructor and set it to `0`. Every time customDevices is read in loadDevices metohd check for how many keys it has and if different than the previous `customDevicesLength` add a debug:

` debug('Loaded', Object.keys(devices).length, 'custom js hass devices configurations')`

Reply:
loadDevices doesn't read the js file because it would be cached by require and it seemed clearer to make them distinct. At first I thought of deprecating the js file, but that would be breaking for someone so I kept the startup read of it.
I guess we can track the change in count yes.